pkg/cli/server/server.go

```diff
@@ -513,7 +513,7 @@ func run(app *cli.Context, cfg *cmds.Server, leaderControllers server.CustomCont
 	}
 
 	agentConfig := cmds.AgentConfig
-	agentConfig.AgentReady = agentReady
+	agentConfig.ContainerRuntimeReady = agentReady
```

Comment:
```suggestion
	agentConfig.ContainerRuntimeReady = containerRuntimeReady
```

Reply:
And also at 
* https://github.com/k3s-io/k3s/blob/1c1746114c94498f8b77e0ba13c43ebe6e0da56a/pkg/cli/server/server.go#L113-L117
* https://github.com/k3s-io/k3s/blob/1c1746114c94498f8b77e0ba13c43ebe6e0da56a/pkg/daemons/config/types.go#L364-L370